libs/wallet/src/TransactionUpdate.graphql

```diff
@@ -0,0 +1,93 @@
+fragment TransactionResultEvent on TransactionResult {
+  partyId
+  hash
+  status: status
+  error
+}
+
+fragment WithdrawalEvent on Withdrawal {
+  id
+  withdrawalStatus: status
+  amount
+  asset {
+    id
+    name
+    symbol
+    decimals
+    status
+    source {
+      ... on ERC20 {
+        contractAddress
+      }
+    }
+  }
+  createdTimestamp
+  withdrawnTimestamp
+  txHash
+  details {
+    ... on Erc20WithdrawalDetails {
+      receiverAddress
+    }
+  }
+  pendingOnForeignChain @client
+}
+
+fragment OrderEvent on Order {
+  type
+  id
+  orderStatus: status
+  rejectionReason
+  createdAt
+  size
+  price
+  timeInForce
+  expiresAt
+  side
+  market {
+    id
+    decimalPlaces
+    positionDecimalPlaces
+    tradableInstrument {
+      instrument {
+        name
+      }
+    }
+  }
+}
+
+fragment DepositEvent on Deposit {
```

Comment:
Deposit events will be emitted off the back of an Ethereum transaction. Are we intending to share these for both the Ethereum transaction store and the Vega transaction store?